resolver.go

```diff
@@ -98,11 +90,45 @@ func PublicResolverAddress(backend bind.ContractBackend) (common.Address, error)
 
 // Address returns the Ethereum address of the domain.
 func (r *Resolver) Address() (common.Address, error) {
-	nameHash, err := NameHash(r.domain)
+	node, err := NameHash(r.domain)
```

Comment:
Please don't change variable names that are already set; it increases the size of the diff and makes it harder to understand.